api/HTMLVideoElement.json

```diff
@@ -119,7 +119,7 @@
               "version_added": "69"
             },
             "opera_android": {
-              "version_added": false
+              "version_added": "59"
```

Comment:
Yay, thanks for updating this! We must have hundreds upon hundreds of entries incorrectly set to false for various Chromium-based browsers...